posthog/queries/funnels/funnel_event_query.py

```diff
@@ -61,6 +61,12 @@ def get_query(self, entities=None, entity_name="events", skip_entity_filter=Fals
                 for group_index in self._column_optimizer.group_types_to_query
             )
 
+            # TODO: prevent possible redundancy here
+            _fields.extend(
```

Comment:
I don't think anything should clash here, since these are all person properties columns, which means their name always starts with `mat_pp_*`, vs `mat_*` for event properties.

Is this what you meant, or something else?

Reply:
@EDsCODE ping on this

Reply:
yeah the prefix is different so we should be ok